pkgs/development/python-modules/types-dateutil/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+}:
+
+buildPythonPackage rec {
+  pname = "types-python-dateutil";
+  version = "2.8.9";
+  format = "setuptools";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "sha256-kPlaa21Pq6NZKH8XosrlEczJ1KvImwGWm9rBGFgVwF0=";
+  };
+
```

Comment:
There are no tests for the stubs. Can you please add `pythonImportsCheck`?

Reply:
Will do.